stdlib/public/SDK/CoreGraphics/CGFloat.swift.gyb

```diff
@@ -690,28 +551,36 @@ public func isnan(_ value: CGFloat) -> Bool { return value.isNaN }
 @available(*, unavailable, message: "use the sign property.")
 public func signbit(_ value: CGFloat) -> Int { return value.sign.rawValue }
 
+// Generic implementation on FloatingPoint as of Swift 4.2
+@available(swift, obsoleted: 4.2)
 @_transparent
 public func modf(_ x: CGFloat) -> (CGFloat, CGFloat) {
   let (ipart, fpart) = modf(x.native)
   return (CGFloat(ipart), CGFloat(fpart))
 }
 
+@available(swift, deprecated: 4.2, renamed: "scalbn")
```

Comment:
It's weird to have something deprecated then provide a rename to something obsoleted in the same version.

Reply:
`scalbn` is obsoleted for concrete types; there's a new `scalbn` that's generic on `BinaryFloatingPoint`.

Reply:
Ah! Thanks for the explanation.

Reply:
Any reason we can't just remove them, then?

Reply:
Not that I know of; ben and max recommended `obsoleted`. If you think removal is viable, I'll make another PR for that.